docs/guide/webpack.md

```diff
@@ -237,3 +237,27 @@ If a custom bundle imports packages that are also imported by the "main" bundle
 ### Deployment
 
 Apostrophe will take care of deploying the output files of custom bundles alongside those generated by the main bundle. Apostrophe will also generate the needed `script` and `link` tags automatically.
+
+## The Webpack cache
+
+Apostrophe uses webpack's built-in support for caching of modules in the filesystem. In normal use you should never have to worry about this, you simply benefit from a speed improvement. However, in certain scenarios you may wish to configure the location of the cache or clear it.
+
+### Clearing the Webpack cache
+
+In cases such as a change to your webpack configuration that Apostrophe cannot automatically detect but which leads to a desired change in the compiled output, you may need to clear the webpack cache. You can do that with this command line task:
```

Comment:
This is very confusing. Did you mean: "and which leads to a desired change not shown in the compiled output"?